apps/app/src/app/pages/(components)/components/(label)/label.page.ts

```diff
@@ -59,7 +59,7 @@ export const routeMeta: RouteMeta = {
 
 			<spartan-page-bottom-nav>
 				<spartan-page-bottom-nav-link href="menubar" label="Menubar" />
-				<spartan-page-bottom-nav-link direction="previous" href="input" label="Input" />
+				<spartan-page-bottom-nav-link direction="previous" href="input-otp" label="Input OTP" />
```

Comment:
I think we need to also update the "Next" button in the input.page.ts

Reply:
Yes its updated on the input too